src/combinatorics/simple_graph/regularity/increment.lean

```diff
@@ -0,0 +1,191 @@
+/-
+Copyright (c) 2022 Yaël Dillies, Bhavik Mehta. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies, Bhavik Mehta
+-/
+import .chunk
+import .energy
```

Comment:
Let's use fully-qualified imports here.

Other than that, let's merge this once you merge master.

bors d+